app/src/views/private/components/header-bar/header-bar.vue

```diff
@@ -112,6 +112,10 @@ export default defineComponent({
 	.title-outer-prepend {
 		display: none;
 
+		:deep(.button) {
+			color: var(--primary);
+		}
```

Comment:
This would change the icon color _everywhere_, rather than just in the couple of pages in settings where it's applicable. The user is normally able to configure the color, so this isn't quite the right solution. As per the original issue, Flows already has the correct icon styling applied, so lets copy over the styling logic used in there to the other pages in settings where it's missing 🙂 

https://github.com/directus/directus/blob/ae6182215965f99f30c1d5b1eeb1a708466e76a4/app/src/modules/settings/routes/flows/flow.vue#L689-L694

Reply:
I tried this and didn't see any change in the applied styling. I also don't see any change on hove with the flows icon either.

Reply:
Looks like there's a small difference in how the buttons are rendered between flows and the other pages. Flows uses:

```html
<v-button class="header-icon" rounded disabled icon>
	<v-icon name="bolt" />
</v-button>
```

whereas the other pages seem to use:

```html
<v-button class="header-icon" rounded disabled icon secondary>
	<v-icon name="list_alt" />
</v-button>
```

the `secondary` prop sets the foreground color to the gray, and has a higher specificity in the context of the page than the custom override. Removing those `secondary` props makes things look like it should 👍🏻